js/src/utils/common.ts

```diff
@@ -1,5 +1,5 @@
-import crypto from "crypto";
+import { v4 as uuidv4 } from "uuid";
 
 export const getUUID = () => {
-  return crypto.randomUUID();
+  return uuidv4();
```

Comment:
@himanshu-dixit Let's verify this once 
PS: We previously used crypto and replaced uuid v4